main_test.go

```diff
@@ -20,6 +20,9 @@ type ComponentTest struct {
 func (f *ComponentTest) InitializeScenario(ctx *godog.ScenarioContext) {
 	authorizationFeature := componenttest.NewAuthorizationFeature()
 	jobsFeature, err := steps.NewJobsFeature()
+	if err != nil {
+		os.Exit(1)
+	}
 	apiFeature := jobsFeature.InitAPIFeature()
 	if err != nil {
 		panic(err)
```

Comment:
Please could you update this to `os.Exit(1)` according to my previous comments

Reply:
yes - done